src/util/create-index.js

```diff
@@ -7,7 +7,7 @@ const os = require('os')
 const COMPONENT_GLOBAL = 'PlotlyExporterComponent'
 const PATH_TO_BUILD = path.join(os.tmpdir(), 'orca-build')
 try {
-  fs.mkdirSync(PATH_TO_BUILD)
+  fs.mkdirSync(PATH_TO_BUILD, 0o777)
```

Comment:
@justin-sleep I'm a bit confused: isn't that supposed to be the default already?

Reply:
The default is 777, but then the default umask is applied:
```JavaScript
> process.umask().toString(8)
'22'
```
leaving the directory with permissions of 755.